Mage.Sets/src/mage/cards/l/LocalizedDestruction.java

```diff
@@ -76,7 +76,10 @@ public boolean apply(Game game, Ability source) {
         if (controller == null) {
             return false;
         }
-
+        if (!controller.chooseUse(this.getOutcome(),
+        "Pay 1 or more {E}? Each creature you control with power equal to the amount of paid this way gains indestructible until end of turn", source, game)) {
+            return false;
```

Comment:
if would `return true` here. Asking player for choices meant the apply had an effect.

Reply:
But they declined to pay so I don't apply any effect. Should I still have the outcome as true?

Reply:
Well my general understanding of the apply return value is that it should be true if anyone happened. Either state change / new info was revealed to a player / a player asked for some choice.

It is not often relevant, and I'm sure a lot of effects return the wrong value on various situations.

Reply:
Good to know!